apps/proto/lib/mix/tasks/namespace/module.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Mix.Tasks.Namespace.Module do
+  alias Mix.Tasks.Namespace
+
+  @apps_to_rewrite Namespace.Task.apps_to_namespace() -- ~w(proto protocol server)a
+
+  def rewrite(module_name) when module_name in @apps_to_rewrite do
```

Comment:
this should have been called `apply`. We're applying the namespace to the module.